### PR TITLE
Feature: Dvd Screensaver, but it's Richard's face

### PR DESCRIPTION
**Pull request by syj67507**
### Summary

In this PR, a new component has been added to display Richard's face that will bounce around like the DVD screensaver. Upon each impact, the image of Richard's face is tinted a new color.

### Changes
- installed the `react-dvd-screensaver` package
- created a `<DvdRichard />` component
- Added a new `<div />` the base so this component is the background of the screen

https://github.com/user-attachments/assets/4f69f3b3-678c-484b-afd7-6985f7baff38

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,19 @@
 import { BirthdayCountdown } from "./components/Countdown";
+import { DvdRichard } from "./components/DvdRichard";
 
 function App() {
 
   return (
     <div>
       <BirthdayCountdown />
+      <div style={{
```
Comment (itscatherinelee) [inline src/App.tsx]: Why did you decide to add a style inline over importing in App.css?

File: src/components/DvdRichard.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    return (
+        <div
+            ref={containerRef}
+            style={{
+                width: window.innerWidth,
+                height: window.innerHeight,
+            }}
+        >
+            <div ref={elementRef} style={{ width: size, height: size, backgroundColor: colors[colorIndex] }}>
```
Comment (itscatherinelee) [inline src/components/DvdRichard.tsx:51]: Same question here

File: src/App.tsx (hunk 1 above)
Comment (syj67507) [reply src/App.tsx]: Mostly just preference, they are functionally the same.

Comment (syj67507) [reply src/App.tsx]: I can move it to the css file

File: src/components/DvdRichard.tsx (hunk 2 above)
Comment (syj67507) [reply src/components/DvdRichard.tsx:51]: This one here uses variables to adjust the css (specifically the size and the color) which is easier to do inline in typescript than doing it in css files

Comment (itscatherinelee) [reply src/components/DvdRichard.tsx:51]: So, if this function ends up making a dvd.css file, would you still keep that inline css there or move it to dvd.css? At what point does it become better practice to create a css file

Comment (syj67507) [reply src/components/DvdRichard.tsx:51]: When looking it up, it looks like css files are slightly (maybe to the point of being negligently) more performative, mainly with the idea that react just has to add a class name to the component versus adding all the style elements to the component. 

I personally just prefer keeping everything in react, which could be inline or just defined as an object in another typescript file and we just pass it to the style prop.

**Review by itscatherinelee — APPROVED** [review]

**Pull request merged**